04-writing_a_DMS_plan.Rmd

```diff
@@ -23,42 +18,66 @@ These following are the major sections of the plan:
 4. **Data preservation, access, timelines** - describe where the data will be made available and when
 5. **Access, distribution, reuse considerations** - describe how you have carefully considered any reasons that might limit sharing
 6. **Oversight** - describe who will manage compliance of the DMS plan
-    
-    
+
+
+## General Requirements
+
+☑️  The plan must be **less than 2 pages**.
+&nbsp;
+
+☑️  Nothing confidential or proprietary is in your plan.  
+
+- _Initially DMS plans will not be made public, but they **may be made public** in the future_.
+
+
 ##  Data Type
 
-A. Types and amount data
+### Types and amount data
+
+☑️  Describe what kind of data will be generated over the course of funding.  
+
+☑️  Describe the expected amounts of data for each kind of data.
 
-- Describe what kind of data will be generated over the course of funding. 
-- Describe the expected amounts of data for each kind of data.
+### Data Sharing
+
+☑️  Describe what data from the project will shared or not shared and justify this decision.
+
+### Metadata, other relevant data, and associated documentation
+
+☑️  List information about metadata.
+&nbsp;
 
-B. Data Sharing
+☑️  List other possibly relevant data.  
 
-- Describe what data from the project will shared or not shared and justify this decision.
+- _This includes data you think is not considered scientific._
 
-C. Metadata, other relevant data, and associated documentation
+☑️  List any relevant information about how you plan to gather the data.
 
-- List information about metadata.
-- List other possibly relevant data (that you think is not considered scientific).
-- List any relevant information about how you plan to gather the data (ex. study protocols and data collection instruments) that you plan to share to make it easier for others to reproduce your work.
```

Comment:
hmmm I do think that the info about how you plan to gather the data might still need to be emphasized

Reply:
nevermind I am blind this morning haha